examples/quickstart/src/main/java/quickstart/ReadmeSnippets.java

```diff
@@ -73,7 +73,7 @@ private void createOAuth2Client() {
             .setAuthorizationMode(AuthorizationMode.PRIVATE_KEY)
             .setClientId("{clientId}")
             .setScopes(new HashSet<>(Arrays.asList("okta.users.read", "okta.apps.read")))
-            .setPrivateKey("/path/to/yourPrivateKey.pem")
+            .setPrivateKey("/path/to/yourPrivateKey.pem") // (or) .setPrivateKey("full PEM payload");
```

Comment:
Please update this file with the various possible ways of setting private key.

Reply:
updated